INZFS.MVC/Views/FundApplication/MultiRadioInput.cshtml

```diff
@@ -0,0 +1,95 @@
+    @Model.SectionInfo.Title
+</a>
+}
+    <span class="govuk-caption-l">Question @Model.QuestionNumber of @Model.TotalQuestions</span>
+
+<form asp-action="save" asp-route-pagename="@Model.PageName" method="post" enctype="multipart/form-data">
+
+    <input type="hidden" name="PageName" value="@Model.PageName" />
+    <input type="hidden" name="Kind" value="@nameof(MultiRadioInputModel)" />
+
+    @if (!ViewData.ModelState.IsValid)
+    {
+        <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary">
+            <h2 class="govuk-error-summary__title" id="error-summary-title">
+                There is a problem
+            </h2>
+            <div class="govuk-error-summary__body">
+                <ul class="govuk-list govuk-error-summary__list">
+                    @foreach (var key in ViewData.ModelState.Keys)
+                    {
+                        var value = ViewData.ModelState[key];
+                        foreach (var error in value.Errors)
+                        {
+                            <li>
+                                <a href="#@Model.FieldName">@error.ErrorMessage</a>
+                            </li>
+                        }
+                    }
+                </ul>
+            </div>
+        </div>
+    }
+
+    <div class="govuk-form-group @(ViewData.ModelState.IsValid ? "" : "govuk-form-group--error")" asp-validation-error-class-for="@Model.Question">
+        <label class="govuk-label govuk-label--l">
+            @if (string.IsNullOrEmpty(Model.TitleQuestion))
+            {
+                @Model.Question
+
+            }
+            else
+            {
+                @Model.TitleQuestion
+            }
+        </label>
+        @Html.Partial("_Hint", Model.Hint)
+        <fieldset class="govuk-fieldset" aria-describedby="timing-hint">
+            <div class="govuk-radios govuk-radios" asp-validation-error-class-for="@Model.Question">
+                @foreach (var Action in Model.Actions)
+                {
+                    <div class="govuk-radios__item">
+                        <input class="govuk-radios__input" id="radio-yes" name="DataInput" type="radio" value="@Action.Value"
+                           @if (Model.DataInput == Action.Value)
+                           {
+                               <text>checked="checked"</text>
+                           }
+                           >
+                        <label class="govuk-label govuk-radios__label">
+                            @Action.Name
+                        </label>
+                    </div>
+                }
+            </div>
+        </fieldset>
+    </div>
+
+
+    @Html.Partial("_Accordion", "")
+    @if (Model.ShowMarkAsComplete)
+    {
+        @Html.Partial("_MarkAsComplete", Model)
+    }
+
+    <div class="edit-item-secondary group">
```

Comment:
Please use newly created partial view here. Refer to the other views for examples

Reply:
Done